### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by E66666666** [E66666666/GTAVManualTransmission#130]
**Describe the bug**
When vehicle is off, vehicle starts when pressing throttle.

**To Reproduce**
- Get in a vehicle.
- Press the assigned key to turn off the engine.
- Press the throttle.

**Expected behavior**
Without "Clutch+Throttle start":
- Engine should stay off with pressing throttle in any gear or clutch position.
- Only the assigned key can be used to restart the engine.

With "Clutch+Throttle start"
- Engine should stay off with pressing throttle in any gear,
- Engine only starts
  - with the assigned key, or
  - when pressing clutch **and** throttle in any gear.

**Screenshots and/or videos**
N/A

**Used versions and additional hardware:**
 - Game version: b2699
 - Mod version: 5.5.0
 - Additional hardware: N/A

**Additional context**
Needs further testing, and reverting to older GTA versions to see if this is new behavior from a recent(ish) patch.

Probably should remove the current allow-option for fakeNeutral and only look at clutch input.

**Comment by E66666666** [comment]
Right.

Grabbed an older backup, and this did not occur on versions <= **v1.0.2372.2**. With game version **v1.0.2545.0** and newer, the character always seems to start when pressing throttle.

A workaround to disable this behavior would be nice.

Also a note: Social Club blocks launching version 1604 and older, so I can only provide compatibility for newer game versions. 2060 was the oldest that launched - but I didn't try to find the actual oldest version still allowed.

**Comment by adelinadragonborn** [comment]
Hey, any update on this issue?

**Comment by ikt32** [comment]
Yeah, use b2372, or provide a patch that fixes this behavior that I can integrate into Manual Transmission.
